jwala-webapp/src/main/webapp/resources/js/react/group-operations/CommandStatusWidget.js

```diff
@@ -7,7 +7,7 @@ var CommandStatusWidget = React.createClass({
     },
     render: function() {
         var self = this;
-
+        this.props.numofRec=0;
```

Comment:
You can add the max.jwala.history property to ApplicationPropertySourceDefinition and that way it gets read from vars.properties and applied in the UI